package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build": "turbo run build --filter=!@atj/infra-cdktf",
     "clean": "turbo run clean",
+    "clean:modules": "find $(git rev-parse --show-toplevel) -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
```

Comment:
Thank you for adding this to scripts! Should we also include `.dist` folders too?

Reply:
Thanks @kalasgarov. I haven't personally encountered scenarios where I've had to recursively delete the `dist` directories to resolve issues. Is that something you've had to do? I can add before merging if so.

Reply:
I usually delete them to solve the caching issues that never ends :) , but I might be overthink about it. You are good to merge the PR without it 👍🏻

Reply:
You're all good, man. I was curious about the use case, but since you find it helpful and had a need for it in the past, I went ahead and added it (`pnpm clean:dist`) and updated the README.